### ISSUE TITLE
YAML schema does not accept `validation.links.absolute_links` value `relative_to_docs`

### ISSUE THREAD

**Issue by Bzly** [squidfunk/mkdocs-material#7811]
### Bug description

[Since version 1.6](https://www.mkdocs.org/user-guide/configuration/#validation-of-absolute-links), the MkDocs setting `validation.links.absolute_links` has had an additional valid value: `relative_to_docs`. 

The YAML schema provided in this repo does not know this value is valid ([link](../blob/1f78e37ddae2b40d054fd0f068085d2cda9d9aeb/docs/schema/validation.json#L46)) - hence my editor throws a wobbly. 

### Related links

- [Reporting a bug](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/)
- [Docs: set up validation](https://squidfunk.github.io/mkdocs-material/creating-your-site/?h=yaml+schema#minimal-configuration)
- [Previous issue regarding the same setting, but different problem](https://github.com/squidfunk/mkdocs-material/issues/6879)

### Reproduction

[9.5.49-yaml-schema-incomplete.zip](https://github.com/user-attachments/files/18188364/9.5.49-yaml-schema-incomplete.zip)

### Steps to reproduce

1. Create a new site with `mkdocs new .`
2. Add the following to your `mkdocs.yml`:
```
validation:
  links:
      absolute_links: relative_to_docs
```

3. Configure your editor to validate mkdocs.yml with the schema provided in this repo
4. It complains

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) using the [built-in info plugin](https://squidfunk.github.io/mkdocs-material/plugins/info/).

**Comment by Bzly** [comment]
Simple PR created #7812.

I hope this implementation is acceptable 🤷

**Comment by niclasheinz** [comment]
I'm closing this issue because its solved with #7812

**Comment by squidfunk** [comment]
Forgot to close it, thanks!
